src/services/services.ts

```diff
@@ -2928,8 +2967,26 @@ namespace ts {
             log("getCompletionData: Is inside comment: " + (new Date().getTime() - start));
 
             if (insideComment) {
-                log("Returning an empty list because completion was inside a comment.");
-                return undefined;
+                // Completion should work inside certain JsDoc tags. For example:
+                //     /** @type {number | string} */
+                // Completion should work in the brackets
+                let insideJsDocTagExpression = false;
+                let tag = getJsDocTagAtPosition(sourceFile, position);

```

Comment:
As the previous token I got was the one in code, not in the comment. Or I can modify the ``getChildren`` and other related functions, if that is preferred solution